### PR TITLE
Improve documentation on cache ttl

### PR DESCRIPTION
**Pull request by shirish93**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[x] Docs
[ ] Other... Please describe:
```

## What is the current behavior?

The docs don't talk about the current default ttl, a naive user might assume they don't need to worry about expiration, and have issues with getting their code to work.

Issue Number: N/A

## What is the new behavior?
Improved documentation.

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

No

### CODE REVIEW

File: content/techniques/caching.md [hunk 1]
```diff
@@ -50,12 +50,20 @@ To add an item to the cache, use the `set` method:
...
-You can also specify a TTL (expiration time) for this specific key, as follows:
+The default expiration time of the cache is 5 seconds.
+
+You can manually specify a TTL (expiration time) for this specific key, as follows:
 
 ```typescript
 await this.cacheManager.set('key', 'value', { ttl: 1000 });
 ```
 
+You can set the expiration time to be effectively infinite as follows:
```
Comment (johannesschobel) [inline content/techniques/caching.md:61]:
```suggestion
You can disable the cache by setting the ttl to `null`:
```

**Review by johannesschobel — CHANGES_REQUESTED** [review]

File: content/techniques/caching.md (hunk 1 above)
Comment (kamilmysliwiec) [reply content/techniques/caching.md:61]: These are two contrary sentences^ (infinite vs disabled)

Comment (johannesschobel) [reply content/techniques/caching.md:61]: damn, you are right.. i meant it the other way, haha..

Comment (johannesschobel) [reply content/techniques/caching.md:61]: corrected my suggestion

File: content/techniques/caching.md [hunk 2]
```diff
@@ -50,12 +50,20 @@ To add an item to the cache, use the `set` method:
...
-You can also specify a TTL (expiration time) for this specific key, as follows:
+The default expiration time of the cache is 5 seconds.
+
+You can manually specify a TTL (expiration time) for this specific key, as follows:
 
 ```typescript
 await this.cacheManager.set('key', 'value', { ttl: 1000 });
 ```
 
+You can disable expiration of the cache by setting the `ttl` to `null`:
```
Comment (kamilmysliwiec) [inline content/techniques/caching.md:61]:
```suggestion
To disable expiration of the cache, set the `ttl` configuration property to `null`:
```

**Comment by kamilmysliwiec** [comment]
LGTM

**Pull request merged**